be/src/agent/agent_task.cpp

```diff
@@ -455,6 +456,31 @@ void run_check_consistency_task(const std::shared_ptr<CheckConsistencyTaskReques
     remove_task_info(agent_task_req->task_type, agent_task_req->signature);
 }
 
+void run_compaction_task(const std::shared_ptr<CompactionTaskRequest>& agent_task_req, ExecEnv* exec_env) {
```

Comment:
What's the difference between this pull request and #21094?

Reply:
This granularity is table and partition

Reply:
`alter table ss compact partition p203303;`
How about add a partition keyword?